### PR TITLE
mvp for parking size

### PR DESCRIPTION
**Pull request by juanii000**
Toast with the hardcoded amount of parking to initialize mvp.

image

image

### CODE REVIEW

File: app/src/main/java/com/example/parkingsystem_kotlin/mvp/presenter/ParkingSizePresenter.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.example.parkingsystem_kotlin.mvp.presenter
+
+import com.example.parkingsystem_kotlin.mvp.contract.ParkingSizeContract
+
+class ParkingSizePresenter(private val model: ParkingSizeContract.Model, private val view: ParkingSizeContract.View): ParkingSizeContract.Presenter {
+
+    override fun onParkingSizeButtonPressed() {
+        val parkingSize = model.getParkingSize()
+        view.showParkingSize(parkingSize)
```
Comment (SartiMau) [inline app/src/main/java/com/example/parkingsystem_kotlin/mvp/presenter/ParkingSizePresenter.kt]: you can do here

view.showParkingSize(model.getParkingSize())

File: app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ActivityView.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.example.parkingsystem_kotlin.mvp.view
```
Comment (SartiMau) [inline app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ActivityView.kt]: put this into a base package inside the view package

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/parkingsystem_kotlin/mvp/presenter/ParkingSizePresenter.kt (hunk 1 above)
Comment (juanii000) [reply app/src/main/java/com/example/parkingsystem_kotlin/mvp/presenter/ParkingSizePresenter.kt]: done

File: app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ActivityView.kt (hunk 2 above)
Comment (juanii000) [reply app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ActivityView.kt]: done, thanks.

**Review by SartiMau — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]
@juanii000  the PR its ok, please rename the MVP classes, because the MVP for the main activity not will be only by setting the parking size. You could rename ir to: ParkingPresenter-ParkingView-ParkingModel

**Review by SartiMau — APPROVED** [review]

**Pull request merged**